plugins/docker-compose/README.md

```diff
@@ -27,4 +27,5 @@ plugins=(... docker-compose)
 | dcl       | `docker-compose logs`    | Show logs of container                                           |
 | dclf      | `docker-compose logs -f` | Show logs and follow output                                      |
 | dcpull    | `docker-compose pull`    | Pull image of a service                                          |
-| dcstart   | `docker-compose start`   | Start a container                                                |
+| dcstart   | `docker-compose start`   | start a container                                                |
```

Comment:
Not to be nitpicky, but there was a pattern of starting with a capital letter. "Start a container"

Reply:
my bad, somehow I didn't even notice I changed it